hbase-server/src/main/java/org/apache/hadoop/hbase/util/RegionMover.java

```diff
@@ -432,6 +437,15 @@ public boolean unload() throws InterruptedException, ExecutionException, Timeout
         }
         // Remove RS present in the exclude file
         stripExcludes(regionServers);
+
+        // Remove decommissioned RS
+        Set<ServerName> decommissionedRS = new HashSet<>(admin.listDecommissionedRegionServers());
+        if (CollectionUtils.isNotEmpty(decommissionedRS)) {
+          regionServers.removeIf(decommissionedRS::contains);
+          LOG.debug("Excluded RegionServers from unloading regions to because they " +
```

Comment:
Can just info level, it is not chatty, and information is useful for client as well.